webpack/assets/javascripts/react_app/components/Editor/components/EditorView.js

```diff
@@ -2,19 +2,6 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import AceEditor from 'react-ace';
 
-import 'brace/mode/ruby';
```

Comment:
just curios, how would be break the foreman-js bundle if we want to? it was shown to be faster in #6719 and wouldn't want to continue loading all vendors as one common chunk?

Reply:
I believe we can implement such a mechanism in `@theforeman/vendor` using the `customModules`:
https://github.com/theforeman/foreman-js/blob/master/packages/vendor-core/lib/modules.js#L64-L67